### PR TITLE
fix ux styling of tabs.

### PR DESCRIPTION
**Pull request by rox163**
### This project uses [semver](semver.org), please check the scope of this pr:

- [ ] #none# - documentation fixes and/or test additions
- [x] #patch# - backwards-compatible bug fix
- [ ] #minor# - adding functionality in a backwards-compatible manner
- [ ] #major# - incompatible API change

# CHANGELOG
fix styling of tabs.

### CODE REVIEW

**Comment by rox163** [comment]
screen shot 2017-03-10 at 4 09 39 pm

**Comment by sglanzer-fire** [comment]
@baconC can you check these and make sure you're happy with the style?

**Comment by sglanzer-fire** [comment]
Approved

[Approved with PullApprove](https://pullapprove.com/ciena-frost/ember-frost-tabs/pull-request/50/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ciena-frost/ember-frost-tabs)

**Comment by rox163** [comment]
pending a quick live demo to Brian.

File: addon/styles/_frost-tab.scss [hunk 1]
```diff
@@ -1,9 +1,8 @@
-$frost-tab-active-border: $frost-color-blue-1;
+$frost-tab-active-border: $frost-color-grey-2;
```
Comment (sglanzer-fire) [inline addon/styles/_frost-tab.scss:1]: I thought this was only certain circumstances - we never have a blue for selection?

Comment (rox163) [reply addon/styles/_frost-tab.scss:1]: based on discussion with @baconC, the line was to be changed to grey-2 for any situation

**Comment by sglanzer-fire** [comment]
@baconC Any reason we would break from a "selection" convention in terms of the colouring here?

**Comment by baconC** [comment]
Not quite sure what you mean by breaking from selection convention?

**Comment by rox163** [comment]
@baconC  I think he means usually selected 'things' have a blue outline/underline, like the list for example. Here, selected/active tab is grey-2

**Comment by sglanzer-fire** [comment]
Activity bump

**Comment by rox163** [comment]
@baconC could you respond to https://github.com/ciena-frost/ember-frost-tabs/pull/50#issuecomment-294930480? last bit of confirmation needed to close this.

**Comment by baconC** [comment]
@rox163 @sglanzer Reason we used the black, is that We have too many blue in the page, we think the grey2 would be better choice to show which tab users are currently on.

**Comment by sglanzer-fire** [comment]
@baconC not sure I agree with that, but let's discuss in person - easier to hash out

**Comment by sglanzer-fire** [comment]
Discussed offline - anything that is a "tab" type will move to the non-blue colouring.  @baconC will update specs for related types and will raise issues to establish consistency

**Comment by sglanzer-fire** [comment]
Approved

[Approved with PullApprove](https://pullapprove.com/ciena-frost/ember-frost-tabs/pull-request/50/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ciena-frost/ember-frost-tabs)

**Pull request merged**